pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -118,6 +118,11 @@ def test_moral_graph_with_edge_present_over_parents(self):
             self.assertTrue(edge in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')] or
                             (edge[1], edge[0]) in [('a', 'b'), ('c', 'b'), ('d', 'a'), ('d', 'b'), ('d', 'e')])
 
+    
+    def test_local_independencies(self):
+        from pgmpy.independencies import Independencies
+        self.assertEqual(self.G.local_independencies('a'),Independencies(['a',['b','c']]))
```

Comment:
@kshitij10496 Since we don't have an `__eq__` (https://docs.python.org/2/reference/datamodel.html#object.__eq__) method in `Independencies`, python doesn't know how to compare it with other `Independencies` objects. So, for solving this you can either compare each attribute of the `Independencies` separately or add an  `__eq__` method in the class.